cli/Makefile

```diff
@@ -66,7 +66,11 @@ fmt-go: $(GO_FILES) go.mod go.sum
 install: | ./package.json
 	pnpm install --filter=cli
 
-e2e: install turbo
+corepack:
+	npm install -g corepack@latest
```

Comment:
This line isn't needed since corepack ships with Node.js

Reply:
The version shipped with Node.js currently includes [this bug](https://github.com/nodejs/corepack/issues/110) which causes the first run of the e2e tests on a machine to fail